File: 19-06-2024_Find_maximum_volume_of_a_cuboid.cpp
class Solution {
  public:
    double maxVolume(double perimeter, double area) {
        // code here
        double h=(perimeter-sqrt(perimeter*perimeter-24*area))/12;
        double l=(perimeter/4-2*h);
        return h*h*l;
    }  
};

File: 19-06-2024_Find_maximum_volume_of_a_cuboid.java
class Solution {
    public double maxVolume(double perimeter, double area) {
        double h = (perimeter - Math.sqrt(perimeter * perimeter - 24 * area)) / 12;
        double l = (perimeter / 4 - 2 * h);
        return h * h * l;
    }
}

